### ISSUE TITLE
[bitnami/clickhouse] Clickhouse - broken labels for persistent volumes

### ISSUE THREAD

**Issue by wiktor2200** [bitnami/charts#15978]
### Name and Version

bitnami/clickhouse 3.1.5

### What architecture are you using?

amd64

### What steps will reproduce the bug?

This bug doesn't exist for default `values.yaml` file, because `labels` are not set by default. But when they are set helm template is broken.
Definition
```
  persistence:
    enabled: true
    storageClass: "ssd"
    labels: 
      env: uat
      app: clickhouse
    annotations: {}
    accessModes:
      - ReadWriteOnce
    size: 10Gi
```

Broken output:
```
  volumeClaimTemplates:
    - metadata:
        name: data
        annotations:
        labels:
          labels:
          app: clickhouse
          env: uat
      spec:
        accessModes:
          - "ReadWriteOnce"
        resources:
          requests:
            storage: "10Gi"
        storageClassName: ssd
```

When `commonLabels` is added problem also exists:
```
  commonLabels: 
    test: test_label
```

Broken output:
```
  volumeClaimTemplates:
    - metadata:
        name: data
        annotations:
        labels:
          test: test_label
          labels:
          app: clickhouse
          env: uat
[...]
```

### Are you using any custom parameters or values?

I'm using values file. I've copied default ones and was modyfing it.

### What is the expected behavior?

There is broken indentation for labels in persistentVolumeClaim definition.

### What do you see instead?

Labels defined with some standards across whole chart to minimize mistakes/typos like this one.

**Comment by wiktor2200** [comment]
Hello @javsalgar! Thanks for bringing in Clickhouse chart into life.
I've already prepared fix for that in #15980, please take a look and approve if you agree with changes.

**Comment by javsalgar** [comment]
Hi!

Thank you so much for opening the ticket! The team will take a look
